internal/cli/terraform.go

```diff
@@ -0,0 +1,113 @@
+package cli
+
+import (
+	"os"
+	"path"
+
+	"github.com/spf13/cobra"
+)
+
+var tfFlags = terraformFlags{
+	OutputDIR: Flag{
+		Name:      "Output Dir",
+		LongForm:  "output-dir",
+		ShortForm: "o",
+		Help: "Output directory for the generated Terraform config files. If not provided, the files will be " +
+			"saved in the current working directory.",
+	},
+}
+
+type (
+	terraformFlags struct {
+		OutputDIR Flag
+	}
+
+	terraformInputs struct {
+		OutputDIR string
+	}
+)
+
+func terraformCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "terraform",
+		Aliases: []string{"tf"},
+		Short:   "Manage terraform configuration for your Auth0 Tenant",
+		Long: "This command facilitates the integration of Auth0 with [Terraform](https://www.terraform.io/), an " +
+			"infrastructure as code tool.",
+	}
+
+	cmd.SetUsageTemplate(resourceUsageTemplate())
+	cmd.AddCommand(generateTerraformCmd(cli))
+
+	return cmd
+}
+
+func generateTerraformCmd(cli *cli) *cobra.Command {
+	var inputs terraformInputs
+
+	cmd := &cobra.Command{
+		Use:     "generate",
+		Aliases: []string{"gen", "export"},
```

Comment:
If we are able to integrate the terraform-exec pkg as well, naming this generate makes more sense, otherwise keeping it as export would fit better. So for now I added both aliases. We can consider keeping just one of these once we explore how viable the terraform-exec pkg is for our use case of generating the config and state as well on behalf of the user.

Reply:
I'd suggest adding a comment so as to not forged to remove the less fitting alias in the future.

Reply:
Also keep in mind that once this goes out in a release, the aliases will be part of the public API, and removing one will be a breaking change. So you might want to consider not adding aliases until it's clear which one should be used.

Reply:
A comment could also be used as a reminder for the above.

Reply:
Added a comment for now in [`6cd31e8` (#792)](https://github.com/auth0/auth0-cli/pull/792/commits/6cd31e8259baf4b803dea2dc1a7cdb576d4bc9ca).